django_react_proj/users-fe/src/components/UsersListRow/UsersListRow.js

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import axios from "axios";
+import { API_URL } from "../../constants";
+import NewUserModal from "../NewUserModal/NewUserModal";
+import ConfirmRemovalModal from "../ConfirmRemovalModal/ConfirmRemovalModal";
+import TaskButton from "../TaskButtonModal/TaskButtonModal";
+import styles from "../UserList/UserList.module.css";
+
+
+const UsersListRow = ({user,resetState}) => {
+    const [taskFlag, setTaskFlag] = useState(
+      false
+  )
+  
+  const ontoggleTaskFlag = () => {
+    setTaskFlag(previous => (
+        !previous
+      ));
+  };
+  
+  const deleteUser = async(id) => {
+    try{
+      await axios.delete(`${API_URL}${id}/`)
+      resetState()
+    } catch (error){
+      console.log(error)
+    }
+  };
+  
+  return(
+    <tr className = {taskFlag ? styles.ButtonTrue : styles.ButtonFalse} key={user.id}>
```

Comment:
I think the `key` should be passed as a prop to a component instead: `users.map(user => <UsersListRow key={user.id} [...] />`